modules/references/client/components/references.api.js

```diff
@@ -0,0 +1,10 @@
+export async function create(reference) {
+  await fetch('/api/references', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json; charset=utf-8'
+      // "Content-Type": "application/x-www-form-urlencoded",
+    },
+    body: JSON.stringify(reference) // eslint-disable-line angular/json-functions
+  });
```

Comment:
Needs a call to `.json()` to fetch the response body, see https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch.